tools/tof_lib_gen/ubuntu_22_04/tembin/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM ubuntu:22.04 AS stage1
+ 
+LABEL maintainer="Robert Budai, <robert.budai@analog.com>"
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install updates to base image
+RUN \
+    apt-get update -y \
+    && apt-get install -y git \
+    && apt-get install -y cmake \
+    && apt-get install -y build-essential \
+    && apt-get install -y apt-utils \
+    && apt-get install -y libssl-dev \
+    && apt-get install wget
+
+
+RUN \
+mkdir tof_lib && cd tof_lib \
+&& mkdir DEBIAN && cd DEBIAN \
+&& echo 'Package:tof-lib\n\
+Version:1.0.1\n\
+Maintainer:Analog Devices Inc\n\
+Architecture:amd64\n\
+Description:ROS_Wrapper_for_ADIs_ToF_SDK' > control
+
+ENV DESTDIR_TOF_LIB=/tof_lib
+
+#Install Glog
```

Comment:
Isn't there a way to put the steps for installing glog in some kind of call/function/etc that would stay in a separate (include) file? And then call it from this docker file? And do the same for other docker files. And also apply this for protobuf and websockets and even the SDK.

Reply:
An other solution would be to 
- make https://github.com/analogdevicesinc/ToF/tree/master/scripts/nxp directory to generic ubuntu by passing as argument the target and use the bash script for update
- or create a similar directory just for ubuntu as the previously mentioned one